install/ui/src/freeipa/widget.js

```diff
@@ -2509,6 +2509,8 @@ IPA.custom_checkboxes_widget = function(spec) {
 
     var that = IPA.checkboxes_widget(spec);
 
+    that.set_value_to_lowercase = spec.set_value_to_lowercase || false;
```

Comment:
When the default is changed, it needs to be changed also in the other widget where it was used: `aci.attributes_widget` 

Without it, following command will break Web UI: `ipa permission-mod AA_foo --attrs=businessCategory`